src/wallet/wallet2.cpp

```diff
@@ -6846,10 +6896,10 @@ std::string wallet2::sign_tx_dump_to_str(unsigned_tx_set &exported_txs, std::vec
 
   // save as binary
   std::ostringstream oss;
-  boost::archive::portable_binary_oarchive ar(oss);
+  binary_archive<true> ar(oss);
   try
   {
-    ar << signed_txes;
+    ::serialization::serialize(ar, signed_txes);
```

Comment:
This uses a return code for primary error reporting, so the serialization-output code needs updating in several places.